<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWidget</class>
 <widget class="QWidget" name="RegistrationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 61, 89);</string>
  </property>
  <widget class="QPushButton" name="cancel_but">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  font-size: 12px;
  text-align: center;
  text-decoration: none;
  outline: none;
  color: #fff;
  background-color: rgb(170, 101, 99);
  border: none;
  border-radius: 15px;
}
QPushButton:hover{
 background-color: rgb(85, 195, 89);
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_but">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  font-size: 12px;
  text-align: center;
  text-decoration: none;
  outline: none;
  color: #fff;
  background-color: #4CAF50;
  border: none;
  border-radius: 15px;
}
QPushButton:hover{
 background-color: rgb(85, 195, 89);
}</string>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tab_widget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
  lw-&gt;LoginWidget</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="user_tab">
    <attribute name="title">
     <string>Individual</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>11</y>
       <width>401</width>
       <height>264</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="amount_label">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Full Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="id_label">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Passport number:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="id_label_2">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>ID number:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="id_label_3">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Phone number:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="id_label_4">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Email:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLineEdit" name="i_name_line">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(173, 173, 173);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhLatinOnly</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passport_line">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="whatsThis">
           <string extracomment="AB1234567"/>
          </property>
          <property name="styleSheet">
           <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(173, 173, 173);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhUppercaseOnly</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="id_line">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="whatsThis">
           <string extracomment="AB1234567"/>
          </property>
          <property name="styleSheet">
           <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(173, 173, 173);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhUppercaseOnly</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="phone_line">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="whatsThis">
           <string extracomment="AB1234567"/>
          </property>
          <property name="styleSheet">
           <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(173, 173, 173);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhUppercaseOnly</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="email_line">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="whatsThis">
           <string extracomment="AB1234567"/>
          </property>
          <property name="styleSheet">
           <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(173, 173, 173);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhUppercaseOnly</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="entity_tab">
    <attribute name="title">
     <string>Entity</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>391</width>
       <height>310</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="amount_label_2">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="id_label_5">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Legal name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="id_label_6">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>PAC</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="id_label_7">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>BIC:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="id_label_8">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Adress:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="id_label_9">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Bank BIC:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QComboBox" name="type_selector">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;Segoe UI&quot;;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="e_name_line">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="whatsThis">
           <string extracomment="AB1234567"/>
          </property>
          <property name="styleSheet">
           <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(173, 173, 173);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhUppercaseOnly</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pac_line">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="whatsThis">
           <string extracomment="AB1234567"/>
          </property>
          <property name="styleSheet">
           <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(173, 173, 173);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhUppercaseOnly</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="bic_line">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="whatsThis">
           <string extracomment="AB1234567"/>
          </property>
          <property name="styleSheet">
           <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(173, 173, 173);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhUppercaseOnly</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="adress_line">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="whatsThis">
           <string extracomment="AB1234567"/>
          </property>
          <property name="styleSheet">
           <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(173, 173, 173);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhUppercaseOnly</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="bank_bic_line">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="whatsThis">
           <string extracomment="AB1234567"/>
          </property>
          <property name="styleSheet">
           <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(173, 173, 173);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhUppercaseOnly</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="warning_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stacked_widget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width:461px;height:361px;</string>
   </property>
  </widget>
  <zorder>stacked_widget</zorder>
  <zorder>cancel_but</zorder>
  <zorder>confirm_but</zorder>
  <zorder>tab_widget</zorder>
  <zorder>warning_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
